src/xscen/biasadjust.py

```diff
@@ -3,11 +3,11 @@
 import logging
 import warnings
 from copy import deepcopy
+from unittest.mock import patch
```

Comment:
Interesting approach. I've been reading a bit about this, and it seems like the `patch` function for `unittest.mock` is meant to be used for temporarily overriding the behaviour of a function. They suggest using it as a decorator or within a context manager *specifically for testing*, i.e.

```python
from unittest.mock import patch
import module

def new_func(*args, **kwargs):
    return "new"

with patch("module.function", new_func):
    assert module.function() == "new"
```

It might be a better idea to create a `contextmanager` for this:

```python
@contextmanager
def override(convert_func):
    original_function = module.function
    try:
        module.function = convert_func
        yield
    finally:
        module.function = original_function

```